test/unit/controllers/InvitationController.test.js

```diff
@@ -76,15 +73,12 @@ describe('InvitationController', () => {
       })
     })
 
-    it('sends invitations', function () {
-      this.timeout(10000)
+    it('sends invitations', () => {
       _.extend(req.params, {communityId: community.id, emails: 'foo@bar.com, bar@baz.com'})
 
       return InvitationController.create(req, res)
       .then(() => {
         expect(Email.sendInvitation).to.have.been.called.exactly(2)
-        expect(apiResult).to.exist
```

Comment:
though it's not really a "unit test", i was intentionally testing the interaction with the SendWithUs API here. i suppose i could give you a test API key for our account which just sends no email, because there are some other tests in the system as well that contact the SendWithUs API.